news/_posts/2020-09-17-bc-break-part-1.md

```diff
@@ -0,0 +1,157 @@
+---
+layout: post
+title: "A deeper look at Backward Compatibility Changes"
+subtitle: "About dependencies, API interfaces, SemVer and Backward Compatibility Changes"
+date:   2020-09-17
+authors: [ MathieuFerment ]
+image: /assets/images/theme/meta-logo-build.png
+icon: icon-code
+tags: [contribution, opensource, development]
+---
+
+In 2011, Marc Andreessen famously "Software is eating the world".
+We can all see nowadays that this prophecy turned out very true: software has definitely eaten this world in this entirety. However software has also evolved in how it is being built.
+
+It becomes harder every day to find an application that does not have dependencies. By dependency we refer to a software component (a package, a library, an application...) that require some specific other software packages to function. For example the package [Doctrine ORM](https://github.com/doctrine/orm) requires the [Symfony Console](https://github.com/symfony/console) package.
+
+Most of the time, using a dependency is a practical mean to avoid reinventing the wheel. Why would you re-implement a PHP HTTP Client when the famous [Guzzle](http://docs.guzzlephp.org/en/stable/) is available, why would you re-build a Logger when the widely used [Monolog](https://github.com/Seldaek/monolog) can be used instead ? The right dependencies allow developers to build faster and focus on the core of their business rather than reinventing components that already exist. This is a great example of the concept of ["dwarfs standing on the shoulders of giants"](Standing on the shoulders of giants) being implemented.
+
+Dependencies can come in many shapes and forms:
+- A SDK that depend on a web API definition, such as GitHub
```

Comment:
... that depends ...

Reply:
Fixed in https://github.com/PrestaShop/prestashop.github.io/pull/777/commits/1a698888de149cecf0416f35ec952bbfdbec852a